pages/tool_assembly/xnat_pic_tool_assembly.md

```diff
@@ -0,0 +1,52 @@
+page_tag: <!---REPLACE THIS with the tag that is used in the tool table--->
+---
+
+## What is XNAT-PIC?
+
+**XNAT for Preclinical Imaging Centers (XNAT-PIC)** consists of a set of tools to store, process and share preclinical imaging studies built on top of the [XNAT](https://www.xnat.org/) imaging informatics platform.
+
+
+ 
+## Who is XNAT-PIC intended for?
+
+Preclinical imaging centers deal with many challenges mostly related to the variety of imaging instrumentation yielding huge volumes of raw data. The current procedures to collect, share and reuse preclinical image data are insufficient, thus revealing an urgent need of standardization in terms of data storage and image processing. XNAT-PIC has been developed to overcome this limitation by extending XNAT’s basic functionalities to meet the needs of preclinical imaging facilities. 
+
+## Which task can be solved with XNAT-PIC?
+
+XNAT-PIC is a set of tools to support preclinical imaging scientists in their data management and processing needs. In particular, it consists of:
+
+1. **MRI2DICOM**, a Magnetic Resonance Imaging (MRI) converter from ParaVision® (Bruker, Inc. Billerica, MA) file format to DICOM standard
+
+2. **XNAT-PIC Uploader** to import multimodal DICOM image datasets to XNAT
+
+3. **XNAT-PIC Pipelines** for analysing single or multiple subjects within the same project in XNAT.
+
+
+## Where can you find training materials and events about XNAT-PIC?
+
+* "Demonstrator 5: XNAT-PIC: expanding XNAT for image archiving and processing to Preclinical Imaging Centers". EOSC-Life website, https://www.eosc-life.eu/d5/
+
+* "Towards sharing and reusing of preclinical image data". Euro-Bioimaging website, https://www.eurobioimaging.eu/news/towards-sharing-and-reusing-of-preclinical-image-data/
+
+* "Data Management: Biological and Preclinical Imaging Perspective". Euro-Bioimaging Virtual Pub, February 12th, 2021. <br>
+[![](https://img.youtube.com/vi/QNiAGuFk53w/0.jpg)](https://youtu.be/QNiAGuFk53w "")
+
+* "XNAT-PIC: expanding XNAT for image archiving and processing to Preclinical Imaging Centers". Demonstrator 5 from Populating EOSC-Life: Success stories for the Demonstrators – Session 1 from January 13, 2021.<br>
+[![](https://img.youtube.com/vi/cpEcfIJJqCo/0.jpg)](https://youtu.be/cpEcfIJJqCo "")
+
+
+## Citation
+
+Please, **XNAT-PIC** by using: <br>
+
+* S. Zullino, A. Paglialonga, W. Dastrù, D. L. Longo, S. Aime. XNAT-PIC: Extending XNAT to Preclinical Imaging Centers, 2021. DOI: https://arxiv.org/abs/2103.02044
+
+## Tools used within XNAT-PIC?
+<!--- Automatically generated table; edit the TAG below to the tag for this page, so that tools that have this page's tag are listed here. You can get the tag for this page from the [list of tags](https://github.com/elixir-europe/rdmkit/blob/master/_data/tags.yml). If it isn't listed there, please raise an issue.--->
+
+{% include toollist.html tag="xnat-pic" %}
```

Comment:
I have added the tools to the tools table: https://docs.google.com/spreadsheets/d/16RESor_qQ_ygI0lQYHR23kbZJUobOWZUbOwhJbLptDE/edit#gid=268211668

Please review. They will be imported automatically. You could consider to register them also in https://bio.tools/register for increased visibility.

Reply:
```suggestion
{% include toollist.html tag="XNAT-PIC" %}
```